src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts

```diff
@@ -111,15 +158,26 @@ class trafficWatch extends viewModelBase {
                 sum += item.ElapsedMilliseconds;
             }
             
-            this.stats.min(min.toLocaleString() + " ms");
-            this.stats.max(max.toLocaleString() + " ms");
+            this.stats.min(this.formatMs(min));
+            this.stats.max(this.formatMs(max));
             this.stats.count(this.filteredData.length.toLocaleString());
-            
-            const avg = countForAvg ? generalUtils.formatNumberToStringFixed(sum * 1.0 / countForAvg, 2) + " ms" : "n/a"; 
+
+            const avg = countForAvg ? this.formatMs((sum * 1.0 / countForAvg)) : "n/a"; 
             this.stats.avg(avg);
         }
     }
-    
+
+    private formatMs(num: number): string {
```

Comment:
I think we already have formatters for this - no need to introduce new one

Reply:
done